tracing/otel_jaeger.go

```diff
@@ -45,23 +45,18 @@ const (
 // - JAEGER_AGENT_HOST
 // - JAEGER_ENDPOINT
 // - JAEGER_SAMPLER_MANAGER_HOST_PORT
-// Otherwise, it will initialize tracing with the OTel auto exporter, as per OTel docs, if any of the following environment variables are set:
-// - OTEL_EXPORTER_OTLP_ENDPOINT
-// - OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
-// - OTEL_TRACES_SAMPLER
-// - OTEL_TRACES_EXPORTER
+//
+// Otherwise, it will initialize tracing with the OTel auto exporter using the environment variables defined in the OTel docs
```

Comment:
A period is missing.
```suggestion
// Otherwise, it will initialize tracing with the OTel auto exporter using the environment variables defined in the OTel docs.
```